### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by ghost**
This is a 3-page website, created using React and changing pages by routing. The requirements are as follow:

Create a website consisting of 3 pages: Home, Calculator, and Quote.
Add your own styles to improve the look and feel.
        In the lesson about styling, we introduced several ways to add styles in a React application:
            React basic styling (inline styles, importing CSS and pre-processed CSS files).
            CSS modules.
            Styled components.
        In this project, you should use React basic styling.
    For every page, you should create a route.
    Make sure that navigation links for all 3 routes are displayed on each page.
    Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  display: flex;
+  justify-content: center;
+}
+
+.li-links {
+  font-size: 8vh;
+  font-weight: bold;
+  color: darkblue;
+  padding: 0 20px;
+}
```
Comment (andersonlebon) [inline src/components/App.css]: - [x] kindly note that one of the requirements says that The general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  Kindly make sure this general layout(Navbar, calculator) is matched in your design.


image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @iKuartz 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.css (hunk 1 above)
Comment (ghost) [reply src/components/App.css]: Yes, you are right.
This is sad, though. I put a lot of effort in trying a different design...
But I'll change it to match the requirements.

**Review by Gambit142 — APPROVED** [review]
Hi @iKuartz 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**